dockerfile/cosmos/localcross.Dockerfile

```diff
@@ -0,0 +1,174 @@
+ARG BASE_VERSION
+FROM --platform=$BUILDPLATFORM golang:${BASE_VERSION} AS build-env
+
+RUN apk add --update --no-cache curl make git libc-dev bash gcc linux-headers eudev-dev
+
+ARG TARGETARCH
+ARG BUILDARCH
+
+RUN if [ "${TARGETARCH}" = "arm64" ] && [ "${BUILDARCH}" != "arm64" ]; then\
+        wget -c https://musl.cc/aarch64-linux-musl-cross.tgz -O - | tar -xzvv --strip-components 1 -C /usr;\
+    elif [ "${TARGETARCH}" = "amd64" ] && [ "${BUILDARCH}" != "amd64" ]; then\
+        wget -c https://musl.cc/x86_64-linux-musl-cross.tgz -O - | tar -xzvv --strip-components 1 -C /usr;\
+    fi
+
+ARG GITHUB_ORGANIZATION
+ARG REPO_HOST
+
+WORKDIR /go/src/${REPO_HOST}/${GITHUB_ORGANIZATION}
+
+ARG GITHUB_REPO
+ARG VERSION
+ARG BUILD_TIMESTAMP
+
+ADD . .
```

Comment:
this is the only change in this file compared to the cosmos `Dockerfile`

Reply:
Ok cool. Does this need to be so "wet"?

Could we export an env var here: https://github.com/strangelove-ventures/heighliner/pull/161/files#diff-d371e466d16c4d92dff29b2806e847c619149cd6de147f1cb9ae7e9511d5da7fR132

and then do an `if`  check here so we don't need two nearly identical files?

Or is there a reason not to do that?